src/rules/mock-rule-builder.ts

```diff
@@ -246,11 +249,11 @@ export default class MockRuleBuilder {
      * before sending requests to be matched. The mocked endpoint
      * can be used to assert on the requests matched by this rule.
      */
-    thenPassThrough(): Promise<MockedEndpoint> {
+    thenPassThrough(forwardToLocation?: string): Promise<MockedEndpoint> {
```

Comment:
Sharing the handler implementation totally makes sense to me, but I think this'd be a little clearer if it was a separate method in the API to actually use it. I'm imagining something like:

```ts
.thenPassThrough();

// and

.thenForwardTo('https://example.com');
```

Would that work for you?